e2etest/bgptests/dump.go

```diff
@@ -12,19 +12,20 @@ import (
 	"github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	"go.universe.tf/e2etest/pkg/frr"
+	frrcontainer "go.universe.tf/e2etest/pkg/frr/container"
 	"go.universe.tf/e2etest/pkg/metallb"
 	clientset "k8s.io/client-go/kubernetes"
 )
 
-func dumpBGPInfo(basePath, testName string, cs clientset.Interface, namespace string) {
+func dumpBGPInfo(basePath, testName string, cs clientset.Interface, namespace string, containers []*frrcontainer.FRR) {
```

Comment:
this change to the testing framework should go in a separate commit.
Also, let's have this accept an "extraContainers" variadic argument

Reply:
I see the the `namespace string` is not being used, should I let it /remove it?

Reply:
maybe in another commit